llvm/lib/Target/RISCV/RISCVInstrInfoV.td

```diff
@@ -976,10 +976,14 @@ multiclass VNCLP_IV_V_X_I<string opcodestr, bits<6> funct6> {
 }
 
 multiclass VSLD_IV_X_I<string opcodestr, bits<6> funct6> {
```

Comment:
Can we pass a bit as an argument to indicate up or down? That seems like a better interface then making assumptions about the name.

Reply:
I agree. It's fixed now.